### ISSUE TITLE
WordPress.Arrays.ArrayDeclarationSpacing.ArrayItemNoNewLine false positive with phpcs:ignore comment

### ISSUE THREAD

**Issue by mundschenk-at** [WordPress-Coding-Standards/WordPress-Coding-Standards#1334]
**Example snippet:**

```php
$foo = [
	'meta_key'   => 'foo', // phpcs:ignore WordPress.VIP.SlowDBQuery.slow_db_query_meta_key
	'meta_value' => 'bar', // phpcs:ignore WordPress.VIP.SlowDBQuery.slow_db_query_meta_value
];
```

**Expected result:** 
No errors

**Actual result:**
Each item in a multi-line array must be on a new line (WordPress.Arrays.ArrayDeclarationSpacing.ArrayItemNoNewLine)

**Comment by jrfnl** [comment]
Hi @mundschenk-at Thanks for reporting this and the example code. I have actually been working on fixing this (and a number of other sniffs which run into the same issue). This will be fixed before version 1.0.0 will be released.

There is a "meta" ticket for this: #1272, so I'd suggest closing this one as a duplicate and continuing the conversation there.

**Comment by jrfnl** [comment]
FYI: I've added your test case to my local WIP branch and ran a test which confirmed that it is already fixed there, so you can be assured that once I pull this your issue will be gone;-)
